setup.py

```diff
@@ -22,7 +30,10 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="The TF Encrypted Authors",
-    author_email="tfencrypted@gmail.com",
+    author_email="contact@tf-encrypted.io",
+    include_package_data=True,
```

Comment:
👍 we should do this for TFE as well (https://github.com/tf-encrypted/tf-encrypted/issues/595)